### DISCUSSION TITLE
A way to define global Env vars in tests, or access request's env vars

### DISCUSSION THREAD

**Discussion by davehouser1** [Kong/insomnia#6517] [Ideas] [answered]
Is there a way to access the request I am testing's env vars with chai syntax?
I tried this but I get the error _"insomnia.getEnvironmentVariable is not a function"_

```
// Gather env vars
const host = insomnia.getEnvironmentVariable("host");
const port = insomnia.getEnvironmentVariable("port");
```

**Comment by subnetmarco** [comment, marked as answer]
Global environments, including their scripting support, [has been released today](https://konghq.com/blog/product-releases/insomnia-9-3-ga) in Insomnia 9.3 GA.
